contrib/buildsystems/engine.pl

```diff
@@ -250,6 +265,7 @@ sub handleCompileLine
         } elsif ($part =~ /\.(c|cc|cpp)$/) {
             $sourcefile = $part;
         } else {
+            print "full line: $line\n";
```

Comment:
> In contrib/buildsystems/engine.pl:
> > @@ -250,6 +265,7 @@ sub handleCompileLine
> >          } elsif ($part =~ /\.(c|cc|cpp)$/) {
> >              $sourcefile = $part;
> >          } else {
> > +            print "full line: $line\n";

> This also seems to be a debug left-over...

No. This is positive error code to capture failures due to upstream 
changes. Without such data the OP would not be able to provide a proper 
error report.

Essentially it is "here is some unhandled rubbish from the makefile 
dry-run that shouldn't be here, I don't know what to do with it".

Perhaps the message could be improved...

--

Philip